vendor/bat-native-ads/src/bat/ads/internal/ml/data/vector_data.cc

```diff
@@ -0,0 +1,104 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <limits>
+#include <map>
+#include <stdexcept>
```

Comment:
See previous comment and link to coding style

Reply:
Done